.github/workflows/bin/spack-reviewers.py

```diff
@@ -33,10 +34,8 @@ def main():
             package = match.group(1)
 
             # add maintainers from packages here
-            result = subprocess.run(
-                ["spack", "maintainers", package], capture_output=True, text=True, check=False
-            )
-            maintainers.update(result.stdout.split())
+            pkg_maintainers = spack.repo.PATH.get_pkg_class(package).maintainers
```

Comment:
Unfortunately, you won't pick up maintainers from base classes anymore.

Granted, there are so few in the base classes anymore (now only `RacketPackage` (9) and `RubyPackage` (18)).

But I still think we should be providing the complete list.

Reply:
Should've tested this first. It looks like we actually DO get the inherited maintainers.

Reply:
I *suppose* it's a moot point.  All packages that inherit from `RubyPackage` don't add maintainers to what is inherited. And all packages that inherit from `RacketPackage` override the inherited maintainer (with the same account).